module/vision/BallDetector/data/config/igus1/BallDetector.yaml

```diff
@@ -40,3 +41,9 @@ clustering:
     # random_subset             a subset of the training samples (random)
     # static_spread             a maximally spread subset of training samples (repeatable)
     # random_spread             a maximally spread subset of training samples (random start)
+
```

Comment:
What's with all this whitespace? Shouldn't your editor remove it all?

Reply:
Copied over directly from the igus, will have a look